optuna/visualization/_parallel_coordinate.py

```diff
@@ -114,7 +139,7 @@ def _get_parallel_coordinate_plot(study: Study, params: Optional[List[str]] = No
             line={
                 "color": dims[0]["values"],
                 "colorscale": "blues",
-                "colorbar": {"title": "Objective Value"},
+                "colorbar": {"title": target_name},
                 "showscale": True,
                 "reversescale": study.direction == StudyDirection.MINIMIZE,
```

Comment:
I'm wondering if it's not user friendlier to fix the color scale to either (any is fine) when `target` is specified. The direction is only associated with the value. As for the matplotlib variant, https://github.com/optuna/optuna/pull/2090 is related.

Reply:
Good catch! I updated PR to display smaller values in darker color when `target` is specified.

Reply:
Verified locally, thanks for the quick fix!